apprentice_teacher.rb

```diff
@@ -26,15 +19,6 @@ def teach_stuff
     response
   end
 
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
-
-  def receive_raise(raise)
-    @salary += raise
-  end
-
   def set_performance_rating(rating)
```

Comment:
This method is very similar between the two teacher classes, how can you refactor?